apps/api/src/app/sessions/entities/session.entity.ts

```diff
@@ -28,8 +35,29 @@ export class Session extends BaseEntity {
   @Field({ description: 'Description' })
   description: string;
 
+  @Column({
+    type: 'simple-enum',
+    enum: SessionStatus,
+    default: SessionStatus.OPEN
+  })
+  @Field(() => SessionStatus, {
+    defaultValue: SessionStatus.OPEN,
+    description: 'Session Status',
+  })
+  @IsEnum(SessionStatus)
+  @IsNotEmpty()
+  status: SessionStatus;
+
+  @Column({
+    default: false
+  })
+  deleted: boolean;
```

Comment:
should we move this to the BaseEntity / new SoftDeleteEntity?

Reply:
Another addition:

Type ORM does support soft delete:
https://typeorm.io/decorator-reference#deletedatecolumn